deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -203,6 +203,15 @@ spec:
       - kind: ServiceMonitors
         name: servicemonitors.monitoring.coreos.com
         version: v1
+      - kind: ScrapeConfigs
+        name: scrapeconfigs.monitoring.coreos.com
+        version: v1alpha1
+      - kind: ServiceMonitors
```

Comment:
This was missing previously, and I'm not 100% sure the effects. My guess is it would affected garbage collection in the operator-sdk, preventing these servicemonitors from being cleaned up automatically when the servicetelemetry object is deleted.